packages/server/src/system/migrations/20231012112401_add_tax_number_column_to_tenants_metadata_table.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.table('tenants_metadata', (table) => {
+    table.string('tax_number')
```

Comment:
the `tax_number` should be under `tax_rates` table, btw application is multi-tenant architecture and has two different type of databases, system database has all the users information and tenants databases each organization has seperate database has everything related to it like invoices, items, bills, etc. that migration should be on tenants level.

To create a new tenant DB migration. You can from the root directory hit `node packages/server/build/commands.js tenants:migrate:make create_new_table` after building `pnpm run build:server`

Reply:
I don't think that's the right place to put it, the tax number is a unique identifier associated with a company. So each tenant should have the ability to use a number. Also it's not actually associated with different tax rates (but depending on region the amount of tax does depend on it I'd suppose). The main reason I'm adding it is because in Australia it's a requirement to have an Australian Business Number (ABN) on any tax invoice.

You are right however that it should be added to all customers. I've actually done that already I'm just cleaning it up.

Reply:
It was my bad, you're absolutely right, I thought the tax number you want to add it under tax rates. Good work buddy.